etc/configuration/default.json

```diff
@@ -808,7 +808,9 @@
       "robot_radius_at_foot_height": 0.2,
       "minimum_robot_radius_at_foot_height": 0.07,
       "ball_obstacle_radius": 0.05,
-      "field_border_weight": 0.15
+      "field_border_weight": 0.15,
+      "line_walking_speed": 0.25,
+      "arc_walking_speed": 0.2
```

Comment:
Oh wait, these are new parameters? Doesn't the walking engine or the odometry know how fast the robot is?

Reply:
As far as I know we have no average known speed for our robots currently. They know their maximum step size but the step planner only outputs one step at a time and does not know how long that step will take.

Reply:
Maybe open an issue to suggest a feature to measure walking speed :)

Reply:
Done